schemas/compat-data.schema.json

```diff
@@ -43,8 +43,20 @@
           "type": "boolean",
           "description": "A boolean value indicating whether or not the implementation of the sub-feature deviates from the specification in a way that may cause compatibility problems. It defaults to false (no interoperability problems expected). If set to true, it is recommended that you add a note explaining how it diverges from the standard (such as that it implements an old version of the standard, for example)."
         },
-        "version_added": { "$ref": "#/definitions/version_value" },
-        "version_removed": { "$ref": "#/definitions/version_value" },
+        "version_added": {
+          "description": "A string the value true, indicating which browser version added this feature, or the value false indicating the feature is not supported, or the value null indicating support is unknown.",
+          "allOf": [
+            { "type": ["string", "boolean", "null"] },
+            { "not": { "enum": ["true", "false", "null"] } }
```

Comment:
Since it's not obvious: the "true", "false" and "null" here merely disallow those *strings*, presumably to help highlight such mishaps in the editor. (The linter would also complain about it.)

Reply:
Yep. History of this is in #485.